example/src/main/java/com/ably/chat/example/MainActivity.kt

```diff
@@ -219,7 +219,7 @@ fun Chat(room: Room, modifier: Modifier = Modifier) {
             onMessageChange = {
                 messageText = it
                 coroutineScope.launch {
-                    room.typing.start()
+                    room.typing.keystroke()
```

Comment:
Worth updating handleSend/handleEdit to call `typing.stop()`?

Reply:
Mm... I didn't work on example app yet. I will check 👍

Reply:
updated